pattern-filter/apps/pattern-filter/pom.xml

```diff
@@ -3,10 +3,10 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.redhat.training</groupId>
-	<artifactId>filter-messages</artifactId>
+	<artifactId>transform-messages</artifactId>
 	<version>0.0.1-SNAPSHOT</version>
-	<name>filter-messages</name>
-	<description>GE: Developing routes filtering messages</description>
+	<name>transform-messages</name>
+	<description>GE: Generating Hello World Messages with the Scheduler Component</description>
 	<properties>
```

Comment:
Please change the dependency to Fuse 7.10 & fix the spacing. We have in this file a mix of tabs & 2 spaces.